### ISSUE TITLE
Problem with accentuated kodi profile

### ISSUE THREAD

**Issue by osmose99** [MediaBrowser/plugin.video.emby#105]
Hi, thanks for this plugin...

I've been getting problems with the emby for kodi plugin when using it on a kodi box with multiple profiles and when those name has accents (ex.: Sébastien, Mélissa). The profile folder use those accent and the emby plugin crash and is unusable.

The only solution that I get is to name those without accent (Sebastien, Melissa). After the profile creation, you can rename the display name with accent. The folder still be without accents so everything work perfectly.

But it could be nice to be able to use it with accentuated names.

Here is the output of the KODI.log
```
21:59:55.403 T:2609525664  NOTICE: EMBY.service -> Delaying emby startup by: 0 sec...
21:59:55.423 T:2609525664  NOTICE: EMBY.service_entry -> ======== START EMBY ========
21:59:55.424 T:2609525664  NOTICE: EMBY.service_entry -> Python Version: 2.7.13 (default, Oct  7 2017, 01:49:17) 
                                            [GCC 6.2.0]
21:59:55.432 T:2609525664  NOTICE: EMBY.service_entry -> Platform: Linux
21:59:55.433 T:2609525664  NOTICE: EMBY.service_entry -> KODI Version: 17.5 Git:5bd45ab
21:59:55.434 T:2609525664  NOTICE: EMBY.service_entry -> EMBY Version: 2.3.56
21:59:55.444 T:2609525664  NOTICE: EMBY.service_entry -> Using plugin paths: True
21:59:55.447 T:2609525664  NOTICE: EMBY.service_entry -> Log Level: 1
21:59:55.450 T:2609525664  NOTICE: EMBY.views -> Clearing nodes properties.
21:59:55.454 T:2609525664  NOTICE: EMBY.service_entry -> Loading existing views...
21:59:55.471 T:2609525664  NOTICE: EMBY.connectionmanager -> Begin ConnectionManager constructor
21:59:55.472 T:2609525664  NOTICE: EMBY.connectionmanager -> Begin connect
21:59:55.473 T:2609525664  NOTICE: EMBY.connectionmanager -> Begin getAvailableServers
21:59:55.474 T:2609525664  NOTICE: EMBY.credentials -> 'ascii' codec can't encode character u'\xe9' in position 34: ordinal not in range(128)
21:59:55.475 T:2609525664  NOTICE: EMBY.credentials -> credentials initialized with: {}
21:59:55.476 T:2609525664  NOTICE: EMBY.connectionmanager -> Begin getConnectServers
21:59:55.811 T:2880431008   ERROR: CecLogMessage - Write: write failed !
21:59:56.485 T:2609525664   ERROR: Previous line repeats 1 times.
21:59:56.486 T:2609525664  NOTICE: EMBY.connectionmanager -> Found Servers: [{u'Name': u'emby', u'Id': u'd9076907bfc7420aa7c5dc90ca11f7d3', u'Address': u'http://192.168.1.20:8096'}]
21:59:56.512 T:2609525664   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.UnicodeEncodeError'>
                                            Error Contents: 'ascii' codec can't encode character u'\xe9' in position 34: ordinal not in range(128)
                                            Traceback (most recent call last):
                                              File "/storage/.kodi/addons/plugin.video.emby/service.py", line 37, in <module>
                                                service = Service()
                                              File "/storage/.kodi/addons/plugin.video.emby/resources/lib/service_entry.py", line 84, in __init__
                                                librarysync.LibrarySync().offline_mode_views()
                                              File "/storage/.kodi/addons/plugin.video.emby/resources/lib/librarysync.py", line 59, in __init__
                                                self.user = userclient.UserClient()
                                              File "/storage/.kodi/addons/plugin.video.emby/resources/lib/userclient.py", line 42, in __init__
                                                self.connectmanager = connectmanager.ConnectManager()
                                              File "/storage/.kodi/addons/plugin.video.emby/resources/lib/connectmanager.py", line 60, in __init__
                                                self.state = self._connect.connect()
                                              File "/storage/.kodi/addons/plugin.video.emby/resources/lib/connect/connectionmanager.py", line 765, in connect
                                                servers = self.getAvailableServers()
                                              File "/storage/.kodi/addons/plugin.video.emby/resources/lib/connect/connectionmanager.py", line 426, in getAvailableServers
                                                self.credentialProvider.getCredentials(credentials)
                                              File "/storage/.kodi/addons/plugin.video.emby/resources/lib/connect/credentials.py", line 75, in getCredentials
                                                self._set(data)
                                              File "/storage/.kodi/addons/plugin.video.emby/resources/lib/connect/credentials.py", line 59, in _set
                                                with open(os.path.join(self.path, 'data.txt'), 'w') as outfile:
                                            UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 34: ordinal not in range(128)
                                            -->End of Python script error report<--
21:59:56.779 T:2609525664 WARNING: CPythonInvoker(9, /storage/.kodi/addons/plugin.video.emby/service.py): the python script "/storage/.kodi/addons/plugin.video.emby/service.py" has left several classes in memory that we couldn't clean up. The classes include: N9XBMCAddon9xbmcaddon5AddonE,N9XBMCAddon4xbmc7MonitorE,N9XBMCAddon9xbmcaddon5AddonE
21:59:57.117 T:2880431008   ERROR: CecLogMessage - Write: write failed !
```

> Using:
> Libreelec 8.2 (Wetek Play)

**Comment by angelblue05** [comment]
Thanks, will check it out.

**Comment by angelblue05** [comment]
I know it's been a while. I think I resolved this issue completely with 3.1.20.

**Comment by osmose99** [comment]
Don't worry about the time it took. I really appreciate. I'll look at it soon.
